scripts/factoids.js

```diff
@@ -39,7 +39,7 @@ module.exports = function(bot) {
       var re = /(.+?)\s(is|are)\s(.+)/;
       var matches = re.exec(text);
       if (!bot.helpers.utils.empty(matches, 1) && !bot.helpers.utils.empty(matches, 2)) {
-        bot.brain.upsertToCollection('factoids', {key: matches[1], channel: to}, {key: matches[1], channel: to, factoid: matches[3], is_are: matches[2]});
+        bot.brain.upsertToCollection('factoids', {key: matches[1].toLowerCase(), channel: to}, {key: matches[1].toLowerCase(), channel: to, factoid: matches[2]});
```

Comment:
The text for the factoid is in `matches[3]`.  The condition should check for `!bot.helpers.utils.empty(matches, 3)` instead and the value for `factoid` should be `matches[3]`.